packages/synapse-react-client/src/lib/containers/markdown/widget/MarkdownButton.tsx

```diff
@@ -21,15 +22,18 @@ export default function MarkdownButton(
     buttonClasses += 'floatright '
   }
   const buttonIsCenterAligned = alignLowerCase === 'center'
-  const buttonVariant = highlight === 'true' ? 'secondary' : 'inherit'
+  const buttonColor = highlight ? 'secondary' : 'neutral'
   const button = (
     <WideButton
       href={widgetParamsMapped.url}
       className={buttonClasses}
       variant="contained"
-      color={buttonVariant}
+      color={buttonColor}
       sx={{
-        '&:hover': { backgroundColor: 'secondary.main', color: 'white' },
+        '&:hover': {
+          backgroundColor: highlight ? undefined : 'secondary.main',
```

Comment:
Don't set the background color in the `highlight === true` case. In that case, MUI created a color that is slightly different, which we don't want to override.